fireant/tests/widgets/test_highcharts.py

```diff
@@ -959,6 +959,221 @@ def test_multi_metrics_multi_axis_line_chart(self):
+                [ElectionOverElection(mock_dataset.fields.timestamp)],
+            )
+        )
+
+        self.assertEqual(
+            {
+                "title": {"text": "Time Series with Hidden Metric and Single Metric"},
+                "xAxis": {"type": "datetime", "visible": True},
+                "yAxis": [
+                    {
+                        "id": "0",
+                        "labels": {"style": {"color": None}},
+                        "title": {"text": None},
+                        "visible": True,
+                    },
+                ],
+                "tooltip": {"shared": True, "useHTML": True, "enabled": True},
+                "legend": {"useHTML": True},
+                "series": [
+                    {
+                        "color": "#DDDF0D",
+                        "dashStyle": "Dash",
+                        "data": [
+                            (820454400000, 7579518.0),
+                            (946684800000, 6564547.0),
+                            (1072915200000, 8367068.0),
+                            (1199145600000, 10036743.0),
+                            (1325376000000, 9491109.0),
+                            (1451606400000, 8148082.0),
+                        ],
+                        "marker": {"fillColor": "#DDDF0D", "symbol": "circle"},
+                        "name": "Votes EoE (Republican)",
+                        "stacking": self.stacking,
+                        "tooltip": {
+                            "valueDecimals": None,
+                            "valuePrefix": None,
+                            "valueSuffix": None,
+                        },
+                        "type": self.chart_type,
+                        "yAxis": "0",
+                    },
+                    {
+                        'color': '#55BF3B',
+                        'dashStyle': 'Dash',
+                        'data': [
+                            (946684800000, 1076384.0),
+                            (1072915200000, 8294949.0),
+                            (1199145600000, 9578189.0),
+                            (1325376000000, 11803106.0),
+                            (1451606400000, 12424128.0),
+                        ],
+                        'marker': {'fillColor': '#DDDF0D', 'symbol': 'square'},
+                        'name': 'Votes EoE (Democrat)',
+                        'stacking': self.stacking,
+                        'tooltip': {'valueDecimals': None, 'valuePrefix': None, 'valueSuffix': None},
+                        'type': self.chart_type,
+                        'yAxis': '0',
+                    },
+                ],
+                "annotations": [],
+                "colors": DEFAULT_COLORS,
+            },
+            result,
+        )
+
+    def test_hidden_ref_with_single_metric_line_chart(self):
+        result = (
+            HighCharts(
+                title="Time Series with Hidden Reference and Single Metric",
```

Comment:
I saw in the code there is a separate change to make it work for pie charts - would be good to add a test for a pie chart as well.